source/codegen/templates/service_helpers.mako

```diff
@@ -659,20 +657,29 @@ ${copy_to_response_with_transform(source_buffer=raw_response_field, parameter_na
 %   elif service_helpers.is_output_array_that_needs_coercion(parameter):
 ${initialize_response_buffer(parameter_name=parameter_name, parameter=parameter)}\
 ${copy_to_response_with_transform(source_buffer=parameter_name, parameter_name=parameter_name, transform_x="x")}\
+%   elif common_helpers.is_struct(parameter) or parameter['type'] == 'ViBoolean[]':
+        convert_to_grpc(${parameter_name}, response->mutable_${parameter_name}());
+%   elif common_helpers.is_string_arg(parameter):
+        response->set_${parameter_name}(${parameter_name});
+%   elif parameter['type'] == 'ViSession':
+        auto session_id = session_repository_->resolve_session_id(${parameter_name});
+        response->mutable_${parameter_name}()->set_id(session_id);
+%   elif parameter['type'] == 'CVIAbsoluteTime':
```

Comment:
I forgot I had to add a special case for `CVIAbsoluteTime` here. It would be nice to get rid of it (maybe tag the field in metadata), but we don't have to do that in this change, it's already pretty big.

Reply:
I was thinking, short term, I would add a "supports_standard_conversion" helper. But yeah, we could also make that more pluggable.